runtime/runtime-params-estimator/Cargo.toml

```diff
@@ -22,7 +22,7 @@ near-crypto = { path = "../../core/crypto" }
 near-vm-logic = {path = "../../runtime/near-vm-logic" , features = ["costs_counting"]}
 near-vm-runner = {path = "../../runtime/near-vm-runner" , features = ["costs_counting", "no_cache", "no_cpu_compatibility_checks", "wasmtime_vm" ]}
 node-runtime = { path = "../../runtime/runtime" , features = ["costs_counting", "no_cache"]}
-near-store = { path = "../../core/store", features = ["no_cache", "single_thread_rocksdb"]}
```

Comment:
I think we still need single_thread_rocksdb

Reply:
Don't think so, this mode of operation doesn't work properly (estimator hangs) and shall not be used.